### DISCUSSION TITLE
Guide on resetting the admin password from CLI

### DISCUSSION THREAD

**Discussion by aruneshdutta** [filebrowser/filebrowser#3387] [General]
Hello kindly guide how to reset the admin password, on running the command inside the container
/filebrowser users update admin --password newpassword I am getting Using database: /database.db and then timeout

**Comment by spidipix** [comment]
Hi, did you solve it? How?

**Comment by chcoomilk** [comment]
one major reason of the timeout error when trying to update the database.db is because the file is in use, make sure you've turned off any filebrowser instance running in the background before editing
